src/app/zap-templates/zcl/data-model/chip/door-lock-cluster.xml

```diff
@@ -199,6 +202,45 @@ limitations under the License.
             <access op="read" role="view" />
             <access op="write" role="administer" />
         </attribute>
+        <!-- Conformance feature [ALIRO] - optional -->
+        <attribute side="server" code="128" define="ALIRO_READER_VERIFICATION_KEY" type="octet_string" length="65" writable="false" optional="true" isNullable="true" apiMaturity="provisional">
+            <description>AliroReaderVerificationKey</description>
+            <access op="read" privilege="administer" />
+        </attribute>
+        <!-- Conformance feature [ALIRO] - optional -->
+        <attribute side="server" code="129" define="ALIRO_READER_GROUP_IDENTIFIER" type="octet_string" length="16" writable="false" optional="true" isNullable="true" apiMaturity="provisional">
+            <description>AliroReaderGroupIdentifier</description>
+            <access op="read" privilege="administer" />
+        </attribute>
+        <!-- Conformance feature [ALIRO] - optional -->
+        <attribute side="server" code="130" define="ALIRO_READER_GROUP_SUBIDENTIFIER" type="octet_string" length="16" writable="false" optional="true" apiMaturity="provisional">
+            <description>AliroReaderGroupSubIdentifier</description>
+            <access op="read" privilege="administer" />
+        </attribute>
+        <!-- Conformance feature [ALIRO] - optional -->
+        <attribute side="server" code="131" define="ALIRO_EXPEDITED_TRANSACTION_SUPPORTED_PROTOCOL_VERSIONS" type="octet_string" array="true" length="16" writable="false" optional="true" apiMaturity="provisional">
```

Comment:
arrays should likely use `entryType` instead of `type`. I am basing this from operational-credentials-cluster.xml , where the type of TrustedRootCertificates shows up correctly in matter files.

type should be 'array'.

Please update this for all arrays.

Reply:
done.